src/lib/adwaitarenderer.h

```diff
@@ -47,7 +47,7 @@ class ADWAITAQT_EXPORT Renderer
 
     static void renderMenuFrame(const StyleOptions &options, bool roundCorners = true);
 
-    static void renderButtonFrame(const StyleOptions &options);
+    static void renderButtonFrame(const StyleOptions &options, bool renderBevel = false);
```

Comment:
This is not ABI compatible change.

Reply:
i wasn't suggesting you should incorporate it. It was a suggestion for a an improved look with a minimal change. so you can see the result. (much better IMO)